app/habit/skeleton.tsx

```diff
@@ -4,6 +4,12 @@ import { Separator } from "@/components/ui/separator";
 import { signOut, useSession } from "next-auth/react";
 import HabitSidebar from "./sidebar";
 import { Button } from "@/components/ui/button";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from "@/components/ui/tooltip";
+import { TooltipProvider } from "@radix-ui/react-tooltip";
```

Comment:
Shouldn't this be imported from "@/components/ui/tooltip"?

Reply:
changed in 095e7b3